crew/connect.go

```diff
@@ -76,6 +81,7 @@ func (t *Tunnel) connect(ctx context.Context) (err error) {
 		t.connInfo.Failed(fmt.Sprintf("failed to establish route: %s", err), "")
 		t.connInfo.Save()
 
+		tracer.Warningf("spn/crew: failed to establish route for %s: %s", t.connInfo, err)
```

Comment:
That sounds like an error but we return nil below

Reply:
The nil return is so that we don't trigger a worker error. As the worker itself did not fail, but something _within_ the worker failed.

Reply:
But that should be up to the caller to decide and not be an assumption that connect() is called solely as a worker?

Reply:
Good point, will rename the function to `connectWorker`, so the context is clear.